pandas/core/window.py

```diff
@@ -374,9 +398,12 @@ def _apply(
         func : str/callable to apply
         name : str, optional
            name of this function
-        window : int/array, default to _get_window()
+        window : int/str, default to _get_window()
```

Comment:
didn't see any passing array to _apply. This seems to be an offset or int same as in `rolling` function